core/src/main/resources/lib/layout/layout.jelly

```diff
@@ -52,6 +52,10 @@ THE SOFTWARE.
       If given, this page is only made available to users that has the specified permission.
       (The permission will be checked against the "it" object.)
     </st:attribute>
+    <st:attribute name="allowedPermissions">
```

Comment:
For consistency with `permission`, all it just `permissions` or perhaps `anyPermission`?